Gemfile

```diff
@@ -1,4 +1,3 @@
 source 'https://rubygems.org'
-ruby '2.2.3'
```

Comment:
It's bulding fine like this: https://federalist.fr.cloud.gov/preview/18f/climate-labs/wslack-remove-223/